plugin/pkg/auth/authorizer/node/graph_populator.go

```diff
@@ -0,0 +1,41 @@
+package node
+
+import (
+	"k8s.io/client-go/tools/cache"
+
+	coreinformers "k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/core/internalversion"
+)
+
+type GraphPopulator struct {
+	graph *Graph
+
+	podInformer              cache.SharedIndexInformer
+	persistentVolumeInformer cache.SharedIndexInformer
+}
+
+func NewGraphPopulator(graph *Graph, pods coreinformers.PodInformer, pvs coreinformers.PersistentVolumeInformer) *GraphPopulator {
+	g := &GraphPopulator{
+		graph:                    graph,
+		podInformer:              pods.Informer(),
+		persistentVolumeInformer: pvs.Informer(),
+	}
+
+	g.podInformer.AddEventHandler(cache.ResourceEventHandlerFuncs{
+		AddFunc:    g.addPod,
+		UpdateFunc: g.updatePod,
+		DeleteFunc: g.deletePod,
+	})
+
+	g.persistentVolumeInformer.AddEventHandler(cache.ResourceEventHandlerFuncs{
+		AddFunc:    g.addPV,
+		UpdateFunc: g.updatePV,
+		DeleteFunc: g.deletePV,
+	})
+
+	return g
+}
+
+func (g *GraphPopulator) Run(stopCh <-chan struct{}) {
+	g.podInformer.GetController().Run(stopCh)
```

Comment:
These 2 lines are likely no-ops (see my comment above, https://github.com/kubernetes/kubernetes/blob/44127c588d24ecabe3db7db66828b181f197a3c7/staging/src/k8s.io/client-go/tools/cache/shared_informer.go#L261-L263, and https://github.com/kubernetes/kubernetes/blob/44127c588d24ecabe3db7db66828b181f197a3c7/staging/src/k8s.io/client-go/tools/cache/shared_informer.go#L161).